[2023-11-26 18:26:38,847] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_district_daily scheduled__2023-11-25T00:00:00+00:00 [queued]>
[2023-11-26 18:26:38,856] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_district_daily scheduled__2023-11-25T00:00:00+00:00 [queued]>
[2023-11-26 18:26:38,857] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-26 18:26:38,857] {taskinstance.py:1357} INFO - Starting attempt 1 of 3
[2023-11-26 18:26:38,857] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-26 18:26:38,870] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_fact_district_daily> on 2023-11-25 00:00:00+00:00
[2023-11-26 18:26:38,876] {standard_task_runner.py:52} INFO - Started process 2074 to run task
[2023-11-26 18:26:38,879] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'final_project_data_covid_etl', 'create_fact_district_daily', 'scheduled__2023-11-25T00:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmp85rp1y5l', '--error-file', '/tmp/tmpx1ptqp7w']
[2023-11-26 18:26:38,879] {standard_task_runner.py:80} INFO - Job 316: Subtask create_fact_district_daily
[2023-11-26 18:26:38,933] {task_command.py:369} INFO - Running <TaskInstance: final_project_data_covid_etl.create_fact_district_daily scheduled__2023-11-25T00:00:00+00:00 [running]> on host 79a606f4be36
[2023-11-26 18:26:39,014] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ricky
AIRFLOW_CTX_DAG_ID=final_project_data_covid_etl
AIRFLOW_CTX_TASK_ID=create_fact_district_daily
AIRFLOW_CTX_EXECUTION_DATE=2023-11-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-25T00:00:00+00:00
[2023-11-26 18:26:39,015] {python.py:173} INFO - Done. Returned value was: None
[2023-11-26 18:26:39,027] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=final_project_data_covid_etl, task_id=create_fact_district_daily, execution_date=20231125T000000, start_date=20231126T182638, end_date=20231126T182639
[2023-11-26 18:26:39,050] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-26 18:26:39,087] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-27 07:13:24,874] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_district_daily scheduled__2023-11-25T00:00:00+00:00 [queued]>
[2023-11-27 07:13:24,885] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_district_daily scheduled__2023-11-25T00:00:00+00:00 [queued]>
[2023-11-27 07:13:24,885] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 07:13:24,886] {taskinstance.py:1357} INFO - Starting attempt 1 of 3
[2023-11-27 07:13:24,886] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 07:13:24,901] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_fact_district_daily> on 2023-11-25 00:00:00+00:00
[2023-11-27 07:13:24,905] {standard_task_runner.py:52} INFO - Started process 227 to run task
[2023-11-27 07:13:24,909] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'final_project_data_covid_etl', 'create_fact_district_daily', 'scheduled__2023-11-25T00:00:00+00:00', '--job-id', '337', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmpfymow3z7', '--error-file', '/tmp/tmp1vcqy7b9']
[2023-11-27 07:13:24,909] {standard_task_runner.py:80} INFO - Job 337: Subtask create_fact_district_daily
[2023-11-27 07:13:24,967] {task_command.py:369} INFO - Running <TaskInstance: final_project_data_covid_etl.create_fact_district_daily scheduled__2023-11-25T00:00:00+00:00 [running]> on host 671693f20ed3
[2023-11-27 07:13:25,035] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ricky
AIRFLOW_CTX_DAG_ID=final_project_data_covid_etl
AIRFLOW_CTX_TASK_ID=create_fact_district_daily
AIRFLOW_CTX_EXECUTION_DATE=2023-11-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-25T00:00:00+00:00
[2023-11-27 07:13:25,036] {python.py:173} INFO - Done. Returned value was: None
[2023-11-27 07:13:25,050] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=final_project_data_covid_etl, task_id=create_fact_district_daily, execution_date=20231125T000000, start_date=20231127T071324, end_date=20231127T071325
[2023-11-27 07:13:25,080] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-27 07:13:25,117] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
